ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -35,7 +35,15 @@
                               (.preventDefault %)
                               (e! (admin-transit-changes/->LoadHashRecalculations)))
                  :icon (ic/content-filter-list)})
-      [:span "Tarkista laskennan eteneminen"]]]))
+      [:span "Tarkista laskennan eteneminen"]]
+     [:a (merge (stylefy/use-style button-styles/negative-button)
```

Comment:
A bit nitpicking: 
There generic nap UI button components are defined in `ote/ui/buttons`, where `/delete`matches to this need style-wise. Might be good to use those whenever possible?